components/twitter/sources/common/followers.js

```diff
@@ -1,4 +1,4 @@
-const twitter = require("../twitter.app");
```

Comment:
@psavkar has a PR (#1135) that may have small conflicts. You may want to either pull those changes into your branch or coordinate with him to resolve any potential merge issues.